<launch>
    <node type="neo_pixel_node.py" pkg="visy_neopixel_pkg" name="neo_pixel_node" output="screen"/>
    <node type="status_bar_node.py" pkg="visy_neopixel_pkg" name="status_bar_node" output="screen"/>
    <node type="light_ring_node.py" pkg="visy_neopixel_pkg" name="light_ring_node" output="screen"/>
    <node type="one_ctrl_node.py" pkg="one_easy_protocol_pkg" name="one_ctrl_node" output="screen"/>

    <arg name="enable_raw" default="false"/>
    <arg name="enable_imv" default="false"/>
    <arg name="camera_id" default="0"/>
    <arg name="camera_frame_id" default="raspicam"/>
    <arg name="camera_name" default="camerav2_1280x960"/>

    <node type="raspicam_node" pkg="raspicam_node" name="raspicam_node" output="screen">
      <param name="private_topics" value="true"/>
      <param name="camera_frame_id" value="$(arg camera_frame_id)"/>
      <param name="enable_raw" value="$(arg enable_raw)"/>
      <param name="enable_imv" value="$(arg enable_imv)"/>
      <param name="camera_id" value="$(arg camera_id)"/>
      <param name="camera_info_url" value="package://raspicam_node/camera_info/camerav2_1280x960.yaml"/>
      <param name="camera_name" value="$(arg camera_name)"/>
      <param name="width" value="1280"/>
      <param name="height" value="960"/>
      <param name="framerate" value="20"/>
      <param name="exposure_mode" value="antishake"/>
      <param name="shutter_speed" value="0"/>
    </node>

</launch>
